CHANGELOG.md

```diff
@@ -53,6 +53,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* (x/group) [#18448](https://github.com/cosmos/cosmos-sdk/pull/18448) Extend group config 
```

Comment:
Can we get this one and the gov one entry in respectively group/CHANGELOG.md and gov changelog instead of the main one?